TthCrstl/config.ini

```diff
@@ -1 +1,76 @@
-humStart=260
\ No newline at end of file
+# Default config.ini
```

Comment:
Why should this font have the default config.ini?

Reply:
If this is the creators intent for a humstart, so be it.